app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+<h2 class="text-lg font-semibold">Log in</h2>
+  <div class="my-4">
+    <%= f.label :email, class: "block text-sm font-medium mb-2" %>
+    <%= f.email_field :email, class: "w-full p-2 border border-gray-400 rounded", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="my-4">
+    <%= f.label :password, class: "block text-sm font-medium" %><br />
+    <%= f.password_field :password, class: "w-full p-2 border border-gray-400 rounded", autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="my-4">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Sign in", class: "bg-blue-400 text-white py-2 px-4 hover:bg-blue-600 rounded cursor-pointer" %>
+  </div>
+  <div class="py-4">
+    <%= render "devise/shared/links" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Please modified your login view style to match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)

![blog_login](https://user-images.githubusercontent.com/55596970/215870491-1488912e-a2ef-4f7e-8dba-dd54785be9b0.png)